### PR TITLE
Push helm operator release versions

### PR DESCRIPTION
**Pull request by aaron7**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -33,12 +33,16 @@ jobs:
               docker login -u "$DOCKER_REGISTRY_USER" -p "$DOCKER_REGISTRY_PASSWORD" quay.io
               docker push "quay.io/weaveworks/flux:$(docker/image-tag)"
             fi
+            if echo "${CIRCLE_TAG}" | grep -Eq "helm-^[0-9]+(\.[0-9]+)*(-[a-z]+)?$"; then
```
Comment (tamarakaufler) [inline .circleci/config.yml]: Shouldn't this be:

"^helm-[0-9]+(\.[0-9]+)*(-[a-z]+)?$"

?

Comment (tamarakaufler) [inline .circleci/config.yml]: When we merge into the master, the helm-operator gets the same tag as flux. Is that what we want?

Comment (squaremo) [reply .circleci/config.yml]: Yes, that's fine. Well, it's fine that they are both named for the commit and branch. It's a bit annoying that changes to the operator code result in a new (but actually unchanged) daemon image, and vice versa.

File: .circleci/config.yml [hunk 2]
```diff
@@ -33,12 +33,16 @@ jobs:
...
 
 workflows:
   version: 2
   build-and-push:
     jobs:
       - build:
           filters:
-            tags:  
+            tags:
               only: /[0-9]+(\.[0-9]+)*(-[a-z]+)?/
```
Comment (squaremo) [inline .circleci/config.yml]: You may have to change this

File: .circleci/config.yml (hunk 1 above)
Comment (tamarakaufler) [reply .circleci/config.yml]: :+1:   I remember we spoke about this problem a while ago.

Comment (aaron7) [reply .circleci/config.yml]: oops, thanks

Comment (squaremo) [reply .circleci/config.yml]: Still needs changing

File: .circleci/config.yml (hunk 2 above)
Comment (squaremo) [reply .circleci/config.yml]: (.. and probably still needs changing)

File: .circleci/config.yml [hunk 3]
```diff
@@ -33,12 +33,16 @@ jobs:
               docker login -u "$DOCKER_REGISTRY_USER" -p "$DOCKER_REGISTRY_PASSWORD" quay.io
               docker push "quay.io/weaveworks/flux:$(docker/image-tag)"
             fi
+            if echo "${CIRCLE_TAG}" | grep -Eq "helm-[0-9]+(\.[0-9]+)*(-[a-z]+)?$"; then
+              docker login -u "$DOCKER_REGISTRY_USER" -p "$DOCKER_REGISTRY_PASSWORD" quay.io
+              docker push "quay.io/weaveworks/helm-operator:$(docker/image-tag | cut -c 5-)"
```
Comment (squaremo) [inline .circleci/config.yml]:
```
$ echo helm-0.1.0-alpha | cut -c 5-
-0.1.0-alpha
```

(also, you may as well use ${CIRCLE_TAG} here)

**Pull request merged**